fetch.bs

```diff
@@ -5744,6 +5751,14 @@ optional boolean <var>forceNewConnection</var> (default false), run these steps:
          <li><p>If <var>request</var>'s <a for=request>mode</a> is "<code>websocket</code>" and
          <var>status</var> is 101, then <a for=iteration>break</a>.
 
+         <li><p>If <var>timingInfo</var>'s
+         <a for="fetch timing info">first interim network-response start time</a> is 0, then set
+         <var>timingInfo</var>'s
+         <a for="fetch timing info">first interim network-response start time</a> to
+         <var>timingInfo</var>'s <a for="fetch timing info">final network-response start time</a>.
```

Comment:
I think this is good, but it doesn't match the description given in https://github.com/w3c/resource-timing/pull/366.

Reply:
Will update the description.

Reply:
I was thinking about this again and how eventually for "`websocket`" we might want to implement the prior step using a shared interim response callback. Based on that I think this step should come first here. It shouldn't have any normative impact, but it will make refactoring seem more straightforward.